schemas/channels/application.schema.json

```diff
@@ -1,41 +1,96 @@
 {
   "meta:license": [
-    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
     "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
   "$id": "https://ns.adobe.com/xdm/channels/application",
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "Application",
-  "type": "object",
   "description":
-    "An application that accepts messages or emit events (Facebook page, Mobile App, ...).\n",
+    "Applications that are installed on desktop or laptop systems.",
+  "meta:extensible": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/channels/channel#/definitions/channel"
+  ],
+  "type": "object",
+  "meta:abstract": false,
   "definitions": {
-    "application": {
+    "channel": {
```

Comment:
This channel stuff seems unrelated to the metrics move.  I would suggest removing these changes from this PR to keep the change clean.

Reply:
The refactoring links this change in to this PR unfortunately. I don't thing I can break it down much more.

@cmathis can we have @jwen-adobe test this branch please and provide feedback?

Reply:
@cdegroot-adobe The meta:enum value you added "https://ns.adobe.com/xdm/channel-types/application": "Desktop Applications" caused schema merging conflict error during the conversion. I think we need to add this enum value to the @type field of all the other channel related schemas to make them consistent??

Reply:
A preview version of the converted schema is here: https://git.corp.adobe.com/experience-platform/sdm-specification/tree/issue265/json/release